ktor-network/nix/src/io/ktor/network/selector/SelectUtilsNix.kt

```diff
@@ -48,7 +48,7 @@ internal actual class SelectorHelper {
     }
 
     actual fun start(scope: CoroutineScope): Job {
-        return scope.launch(CoroutineName("selector")) {
+        return scope.launch {
```

Comment:
Could you tell me why we don't need the name? I thought it can help in debug

Reply:
I moved it to https://github.com/ktorio/ktor/pull/4671/files#diff-675619f9bef0e1f4942b7d0716aa05e1f3c09dad8aa1c0c79004d931e6dfd5a5R12 to keep it a bit more consistent to where it is defined on JVM. So it is not removed.

Reply:
It does indeed help with naming on supported platforms so it is important to keep.